.github/workflows/build.yml

```diff
@@ -22,10 +22,10 @@ jobs:
       GOARCH: amd64
       GOPROXY: https://proxy.golang.org
       JDK_VER: 13.0.x
-      DAPR_RUNTIME_VER: 0.10.0-rc.0
+      DAPR_RUNTIME_VER: 0.11.0-rc.2
       DAPR_INSTALL_URL: https://raw.githubusercontent.com/dapr/cli/e7c9a643dfefbcfff0c2c26c12029259e6e81180/install/install.sh
```

Comment:
Does this need to be

https://raw.githubusercontent.com/dapr/cli/release-0.11/install/install.sh ?

Reply:
It does not need to be. I will change in a separate PR.